data_model/src/lib.rs

```diff
@@ -85,18 +87,10 @@ impl ValidationError {
     Serialize,
     IntoSchema,
 )]
+#[repr(transparent)]
 pub struct Name(String);
```

Comment:
As a side note. I'm 100% confident that we can get away with fixed immutable strings. We could save a machine word, `capacity`, for each `Name` if we use `str` instead of `String`.

Reply:
Created #1985

Reply:
ok, I'm closing this now that we have issue